src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -3255,6 +3255,8 @@ private static bool TryGetNumberValue(string strNum, bool hex, bool real, Number
                 {
                     NumberStyles style = NumberStyles.AllowLeadingSign | NumberStyles.AllowDecimalPoint |
                                          NumberStyles.AllowExponent;
+
+                    // Decimal parser does not accept hex literals, and 'd' is a valid hex character, so will never be read as Decimal literal
```

Comment:
We could add sample.

Reply:
In what sense do you mean? Just an example?

Reply:
Yes, like `0x10d`.

Reply:
Will do! 😄

Reply:
Closed.